client/battle/BattleWindow.cpp

```diff
@@ -204,6 +204,16 @@ void BattleWindow::keyPressed(EShortcut key)
 	InterfaceObjectConfigurable::keyPressed(key);
 }
 
+void BattleWindow::clickPressed(const Point & cursorPosition)
+{
+	if (owner.openingPlaying())
+	{
+		owner.openingEnd();
+		return;
+	}
+	InterfaceObjectConfigurable::clickPressed(cursorPosition);

```

Comment:
shouldn't order be reversed (move before "if") so single click won't trigger battlefield action after opening ends?

Reply:
It is exact the same logic as for "keyPressed".

I don't understand what you mean. After opening ends the function ignores the action with "return".

Reply:
at yeah, nevermind